requirements.txt

```diff
@@ -0,0 +1,12 @@
+--index-url https://pypi.python.org/simple/
+
+-e .
```

Comment:
I would perhaps remove this from the requirements, since the requirements is typically about requiring other packages.

Reply:
Yes, casual user does not need editable mode (-e .) and a dev can add it by himself.